layouts/partials/sidebar.html

```diff
@@ -1,19 +1,22 @@
 <aside class="l-sidebar">
 
   <div class="sections sidebar">
-    <section class="sidebar">
-      <header>LATESTS</header>
-      <div>
-        <div class="articles sm">
-          {{ range $i, $p := (first 10 .Site.Pages) }}
-          {{ .Render "li_sm" }}
-          {{ end }}
-        </div>
-      </div>
-    </section>
+    {{ $order := .Site.Params.SidebarOrder | default ( slice "latests" "categories" "tags" ) }}
+
+    {{ range $i, $key := $order }}
+
+    {{ if eq ( $key | lower ) "latests" }}
+    {{ partial "latests.html" $ }}
+    {{ end }}
+
+    {{ if eq ( $key | lower ) "categories" }}
+    {{ partial "taxonomy.html" (dict "context" $ "key" $key "value" $.Site.Taxonomies.categories) }}
+    {{ end }}
+
+    {{ if eq ( $key | lower ) "tags" }}
+    {{ partial "taxonomy.html" (dict "context" $ "key" $key "value" $.Site.Taxonomies.tags) }}
+    {{ end }}
```

Comment:
hmm 🤔 

I think this implementation is pretty complex.

I guess this design is simple and extendable more than it, How do you think?

sidebar.html
``` html
<div class="sections sidebar">
  {{ partial "latest.html" }}
  {{ partial "categories.html" }}
  {{ partial "tags.html" }}
</div>
```

* layouts/partials/latests.html
* layouts/partials/xxxx.html
* layouts/partials/xxxx.html
* layouts/partials/xxx `// added here if you want`

Override `layouts/partials/sidebar.html` when want to change content order.
https://gohugo.io/templates/lookup-order/

But `taxonomy.html` and `tags.html` could not be DRY 😢

Reply:
Oh, thanks! 😆 

I created new pull-request following your ideas!
And I made taxonomies DRY to using taxonomy.html inner categories and tags 🎉 

Please check #22 🙏